addons/website_sale/views/snippets/snippets.xml

```diff
@@ -364,6 +320,50 @@
                 <we-button class="o_we_bg_success" data-name="o_wsale_add_extra_images" data-add-images="true" data-no-preview="true">Add</we-button>
                 <we-button class="o_we_bg_danger" data-name="o_wsale_clear_extra_images" data-clear-images="true" data-no-preview="true">Remove all</we-button>
             </we-row>
+            <we-row string="Cart" class="o_we_full_row" data-name="o_wsale_buy_now_opt">
+                <we-button
+                    title="Buy Now"
+                    class="o_we_buy_now_btn"
+                    data-customize-website-views="website_sale.product_buy_now"
+                    data-no-preview="true"
+                    data-reload="/"
+                >
+                    <i class="fa fa-fw fa-bolt"/>
+                    Buy Now
+                </we-button>
+                <we-button
+                    title="Select Quantity"
+                    class="o_we_buy_now_btn"
+                    data-customize-website-views="website_sale.product_quantity"
+                    data-no-preview="true"
+                    data-reload="/"
+                >
+                    Quantity
+                </we-button>
+            </we-row>
+            <div data-name="o_wsale_actions_opt" class="d-none">
+                <we-row string="Actions" class="o_we_full_row" data-name="o_we_actions_opt"/>
+            </div>
```

Comment:
```suggestion
                <we-row string="Actions" class="o_we_full_row" data-name="o_wsale_actions_opt" class="d-none"/>
```

why need div? :thinking: can't we just apply d-none class on we-row and remove add element added

Reply:
I tried applying, but it doesn't seem to work when I directly add `d-none` class inside `we-row`